jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -187,8 +183,9 @@ MavenProject getProject() {
     return Preconditions.checkNotNull(project);
   }
 
+  @Nullable
```

Comment:
The Gradle counterpart ensure non-null. Can we be consistent?

Reply:
Ok